<?xml version="1.0"?>
<launch>

	<group ns="uav0">
		<node pkg="fw_control" name="tracking_uav_control" type="uav_control" output="screen" />
	</group>
</launch>

